index.html

```diff
@@ -1734,28 +1781,77 @@ <h3>Document Encoding</h3>
             for processing, in which case the serialisation requirements do not apply.</p>
       </section>
       <section>
-        <h3>Foreign Elements and Attributes</h3>
+        <h3>Unrecognised Elements and Attributes</h3>
         <p>
           A <a>DAPT Document</a> MAY contain elements and attributes that are neither specifically permitted nor forbidden by a profile.
-        </p>
-        <p class ="note">
+          Those elements and attributes could be in a namespace used by DAPT or TTML, or some other namespace.</p>
+        <p><dfn>Foreign vocabulary</dfn> consists of the set of elements and attributes whose namespace
+          is not one of the namespaces listed in <a href="#namespaces"></a> and
+          the set of attributes in the global namespace that are not otherwise defined in DAPT or in [[TTML2]].</p>
+        <div class="note">
+          <p>The requirements in this section are intended to facilitate forwards and backwards compatibility,
+            specifically to permit:
+          </p>
+          <ul>
+            <li>DAPT processors targeted at one version of the specification to process <a>DAPT documents</a> that
+              include vocabulary or semantics defined in future versions,
+              albeit without supporting the later features;</li>
+            <li>DAPT processors targeted at one version of the specification to process <a>DAPT documents</a>
+              authored for an earlier version with similar or identical behaviour to a DAPT processor
+              targeted at that earlier version.</li>
+          </ul>
+        </div>
+        <p class="note">
           <a>DAPT Documents</a> remain subject to the content conformance requirements specified at Section 3.1 of [[TTML2]].
-          In particular, a <a>DAPT Document</a> can contain elements and attributes not in any TT namespace, i.e. in foreign namespaces, since such elements and attributes are pruned by the algorithm at Section 4 of [[TTML2]] prior to evaluating content conformance.
+          In particular, a <a>DAPT Document</a> can contain <a>foreign vocabulary</a>,
+          since such elements and attributes are pruned by the algorithm at Section 4 of [[TTML2]] prior to evaluating content conformance.
         </p>
```

Comment:
Not sure I understand - just to check, do you mean:

* §6.1 before this PR, i.e. [6.1 Conformance of DAPT Documents](https://www.w3.org/TR/dapt/#conformance-of-dapt-documents) or
* §6.1 in this PR, i.e. [6.1 Early identification of non-conformant documents](https://pr-preview.s3.amazonaws.com/w3c/dapt/pull/216.html#early-identification-of-non-conformant-documents)

I'm not sure why it would be in either of those, if you could explain more?

Reply:
Sorry, I meant 6.1 before the PR because it deals with conformance.

Reply:
Done in d3e35b9.